tests/ui_tests/app/actions_page.py

```diff
@@ -0,0 +1,31 @@
+from tests.ui_tests.app.locators import ActionPageLocators
+from tests.ui_tests.app.pages import BasePage
+import allure
+
+
+class ActionPage(BasePage):
+    """Class for action page."""
+
+    def __init__(self, driver, url=None):
+        super().__init__(driver)
+        if url:
+            self.get(url, "action")
+
+    @allure.step('Open run action popup')
+    def open_run_action_popup(self):
+        self._wait_element_present(ActionPageLocators.action_run_button, timer=15).click()
+        self._wait_element_present(ActionPageLocators.run_action_popup)
+
+    @allure.step('Check if verbose checkbox is displayed on popup')
+    def check_verbose_chbx_displayed(self):
+        self.open_run_action_popup()
+        return self.driver.find_element(
+            *ActionPageLocators.ActionRunPopup.verbose_chbx).is_displayed()
```

Comment:
Why unpacking here? And fix formatting
```suggestion
        return self.driver.find_element(
            *ActionPageLocators.ActionRunPopup.verbose_chbx
        ).is_displayed()
```